pallets/baby-liminal/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
```

Comment:
Can you put as a comment explanation why we need this? I've never seen it anywhere.

Reply:
because compiler asked. there's one in runtime